acos_client/v21/action.py

```diff
@@ -39,5 +39,8 @@ def reload(self, write_memory=False, **kwargs):
     def activate_and_write(self, partition, **kwargs):
         write_format = "active-partition {0}\r\nwrite memory\r\n"
         post_body = write_format.format(partition)
-
-        return self._request("POST", "cli.deploy", params=None, payload=post_body, **kwargs)
+        # Request raises an exception when the "maybe error" is returned.
+        try:
+            return self._request("POST", "cli.deploy", params=None, payload=post_body, **kwargs)
+        except acos_errors.ACOSException:
```

Comment:
If this was v30 I'd want to see a further check on the string, but as it's v21, +1